dashboard/config/scripts_json/coursee-2021.script_json

```diff
@@ -4464,22 +4464,22 @@
       "properties": {
         "progression": "Practice",
         "level_keys": [
-          "courseE_farmer_ramp12g_2021"
+          "courseE_farmer_while1_2021"
         ]
       },
       "named_level": null,
       "bonus": null,
       "seeding_key": {
         "script_level.level_keys": [
-          "courseE_farmer_ramp12g_2021"
+          "courseE_farmer_while1_2021"
         ],
         "lesson.key": "Conditionals",
         "lesson_group.key": "ramp_up",
         "script.name": "coursee-2021",
         "activity_section.key": "c6d24990-be45-4b52-b095-48004fcaca4a"
       },
       "level_keys": [
-        "courseE_farmer_ramp12g_2021"
+        "courseE_farmer_while1_2021"
```

Comment:
here and above, the diffs represent changes to the ScriptLevel. I am inferring that `courseE_farmer_ramp12g_2021` should be replaced with `courseE_farmer_while1_2021` here based on what I see in the diff farther down.